pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.ios.kt

```diff
@@ -1,9 +1,106 @@
\ No newline at end of file
+actual interface ManageChannelMembers : Endpoint<PNMemberArrayResult>
+
+@OptIn(ExperimentalForeignApi::class)
+class SetChannelMembersImpl(
+    private val pubnub: PubNubObjC,
+    private val channel: String,
+    private val uuids: List<MemberInput>,
+    private val limit: Int?,
+    private val page: PNPage?,
+    private val filter: String?,
+    private val sort: Collection<PNSortKey<PNMemberKey>>,
+    private val includeCount: Boolean,
+    private val includeCustom: Boolean,
+    private val includeUUIDDetails: PNUUIDDetailsLevel?
+): ManageChannelMembers {
+    override fun async(callback: Consumer<Result<PNMemberArrayResult>>) {
+        pubnub.setChannelMembersWithChannel(
+            channel = channel,
+            uuids = uuids.map { PubNubUUIDMetadataObjC(id = it.uuid, custom = it.custom?.value, status = it.status) },
+            limit = limit?.let { NSNumber(it) },
+            page = createPubNubHashedPage(from = page),
+            filter = filter,
+            sort = sort.map { it.key.fieldName },
+            includeCount = includeCount,
+            includeCustom = includeCustom,
+            includeUUIDFields = includeUUIDDetails == PNUUIDDetailsLevel.UUID,
+            includeUUIDCustomFields = includeUUIDDetails == PNUUIDDetailsLevel.UUID_WITH_CUSTOM,
+            onSuccess = callback.onSuccessHandler3 { data, totalCount, next ->
+                PNMemberArrayResult(
+                    status = 200,
+                    data = filteredList(data) { rawValue: PubNubMembershipMetadataObjC -> createPNMember(rawValue) },
+                    totalCount = totalCount?.intValue,
+                    next = next?.end()?.let { hash -> PNPage.PNNext(pageHash = hash) },
+                    prev = next?.start()?.let { hash -> PNPage.PNPrev(pageHash = hash) }
+                )
+            },
+            onFailure = callback.onFailureHandler()
+        )
+    }
+}
+
+@OptIn(ExperimentalForeignApi::class)
+class RemoveChannelMembersImpl(
+    private val pubnub: PubNubObjC,
+    private val channel: String,
+    private val uuids: List<String>,
+    private val limit: Int?,
+    private val page: PNPage?,
+    private val filter: String?,
+    private val sort: Collection<PNSortKey<PNMemberKey>>,
+    private val includeCount: Boolean,
+    private val includeCustom: Boolean,
+    private val includeUUIDDetails: PNUUIDDetailsLevel?
+): ManageChannelMembers {
+    override fun async(callback: Consumer<Result<PNMemberArrayResult>>) {
+        pubnub.removeChannelMembersWithChannel(
+            channel = channel,
+            uuids = uuids,
+            limit = limit?.let { NSNumber(it) },
+            page = createPubNubHashedPage(from = page),
+            filter = filter,
+            sort = sort.map { it.key.fieldName },
+            includeCount = includeCount,
+            includeCustom = includeCustom,
+            includeUUIDFields = includeUUIDDetails == PNUUIDDetailsLevel.UUID,
```

Comment:
and here

Reply:
I think you made a mistake in this one, check this and line below (`||` is in wrong line)

Reply:
Indeed, thx for catching this. Fixed